app/lib/search/mem_index.dart

```diff
@@ -313,9 +313,10 @@ class InMemoryPackageIndex {
     _adjustedOverallScores = _documents.map((doc) {
       final downloadScore = doc.downloadScore ?? 0.0;
       final likeScore = doc.likeScore ?? 0.0;
-      final combinedScore = (downloadScore + likeScore) / 2;
-      final points = doc.grantedPoints / math.max(1, doc.maxPoints);
-      final overall = combinedScore * 0.5 + points * 0.5;
+      final popularityScore = (downloadScore + likeScore) / 2;
+      final pointScore =
+          math.sqrt(doc.grantedPoints / math.max(1, doc.maxPoints));
```

Comment:
This looks wrong. `math.max(1, doc.maxPoints)` will always be one (unless you got exactly 0 points?

Should it not be

```
          math.max(1, math.sqrt(doc.grantedPoints / doc.maxPoints));
```

Which then simplifies to:

```suggestion
          math.sqrt(doc.grantedPoints / doc.maxPoints);
```

assuming `grantedPoints <= maxPoints`.

Reply:
`math.max(1, doc.maxPoints)` prevents division by zero (in case we have data for whatever reason).

we could have an additional check that `grantedPoints/maxPoints` doesn't go over 1, adding that too.

Reply:
Ah - I was reading `min` - sorry for my confusion :)